src/benchmarklib/table_generator.cpp

```diff
@@ -112,11 +113,15 @@ std::shared_ptr<Table> TableGenerator::generate_table(
   auto allocator_value_segment_int = PolymorphicAllocator<ValueSegment<int>>{};
   auto allocator_chunk = PolymorphicAllocator<Chunk>{};
   auto allocator_int = PolymorphicAllocator<int>{};
+#if HYRISE_NUMA_SUPPORT
+  auto node_id = 0;
+#endif
 
   for (ChunkID chunk_index{0}; chunk_index < num_chunks; ++chunk_index) {
 #if HYRISE_NUMA_SUPPORT
     if (numa_distribute_chunks) {
-      auto memory_resource = NUMAPlacementManager::get().get_next_memory_resource();
+      auto memory_resource = Topology::get().get_memory_resource(node_id);
+      node_id = (node_id + 1) % Topology::get().nodes().size();
```

Comment:
Is round robin really what we want to do? Does this have any benefits?

Reply:
> Does this have any benefits?

Questionable so far. But that is only applied if `numa_distribute_chunks=True`. Or do you want to get rid of that option entirely?

Reply:
Is it ever set? That in itself has only limited value.

Reply:
In the code in the repository, not that I know of. It was introduced as an option to play around with when using the table generator in a playground context.

Reply:
I don't really mind either way. It's only a couple of lines in this file and has no external implications.